docs/usage.txt

```diff
@@ -63,6 +63,33 @@ Result:
         <link rel="preload" href="/static/CACHE/js/d01466eb4fc6.js" as="script" />
 
 
+
+ES5 nomodule & ES6 type="module"
+
+Adding the ``nomodule`` attribute will generate be passed attribute to the script tag for the compressed resource in the template.
+Adding the ``module`` parameter will generate the script tag with type="module" for the compressed resource in the template:
+
+    .. code-block:: django
+
+    {% compress js %}
+        <!-- ES5 must be in separate compress (split types) -->
+        <script nomodule async src="static/js/es5.js"></script>
+    {% endcompress %}
+
+    {% compress js module %}
```

Comment:
I think it would be better if this supported the `type="module"` syntax instead of the additional templatetag parameter. Then i would also worry less if it still worked properly with COMPRESS_ENABLED = False.